src/main/java/org/example/learningsystem/btp/destination/config/DestinationServiceProperties.java

```diff
@@ -0,0 +1,25 @@
+package org.example.learningsystem.btp.destination.config;
+
+import jakarta.validation.constraints.NotNull;
+import lombok.Getter;
+import lombok.Setter;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.stereotype.Component;
+import org.springframework.validation.annotation.Validated;
+
+@ConfigurationProperties(prefix = "destination-service")
+@Component
+@Getter
+@Setter
+@Validated
+public class DestinationServiceProperties {
+
+    @NotNull
+    private String uri;
+    @NotNull
+    private String url;
```

Comment:
it's better to rename one these variables, as from developer's pov who will use this class, it's not obvious which of them to use to get token and which for sending requests:
```
url -> baseTokenUrl or tokenUrl
```